### ISSUE TITLE
Adding sqoop to the sqoop2 group fails without --append-cluster-name

### ISSUE THREAD

**Issue by dmtucker** [Isilon/isilon_hadoop_tools#27]
[This](https://github.com/Isilon/isilon_hadoop_tools/blob/4df24d13973c44de6211a5a1aa3ffda140a06187/isilon_create_users.sh#L298) doesn't seem to work if `--append-cluster-name` is not used!
When it *is* used:
```
dtucker-onefs-1# sqp=`grep sqoop-clustername cdh-clustername.System.group`
dtucker-onefs-1# echo $sqp                                                
sqoop-clustername:x:1022:
dtucker-onefs-1# sed -i .bak /$sqp/d cdh-clustername.System.group
dtucker-onefs-1# 
```
When it *isn't* used:
```
dtucker-onefs-1# sqp=`grep sqoop cdh.System.group` 
dtucker-onefs-1# echo $sqp                                                
sqoop:x:1022:
sqoop2:x:1023:,sqoop
dtucker-onefs-1# sed -i .bak /$sqp/d cdh.System.group
sed: 1: "/sqoop:x:1022:
sqoop2:x ...": unterminated regular expression
```

**Comment by bonibruno** [comment]
Fixed.  See PR bonibuno-patch3.

**Comment by bonibruno** [comment]
Fixed in PR #32, should be pulled into the master repo soon.
